ctapipe/instrument/trigger.py

```diff
@@ -0,0 +1,97 @@
+from ctapipe.containers import ArrayEventContainer
+from ctapipe.core import TelescopeComponent
+from ctapipe.core.traits import Integer, IntTelescopeParameter
+
+
+class SoftwareTrigger(TelescopeComponent):
+    """
+    A stereo trigger that can remove telescope events from subarray events.
+
+    This class is needed to correctly handle super-arrays as simulated for
+    CTA and still handle the LST hardware stereo trigger and the normal stereo
+    trigger correctly.
+
+    LSTs have a hardware trigger, that requires at least two LSTs to have a trigger
+    in each event. Thus, after selecting a subarray, it is needed to
+    - Remove single LST telescope events from events
+    - Ignore events with only 1 telescope after this has been applied
```

Comment:
This can do even more, right? So this is basically an example of what to do with this SoftwareTrigger?  My proposal is to add `Example\n-------` before the last paragraph.

Reply:
Perhaps we should explain that In the current sims, the trigger is not really correctly simulated due to the use of super-arrays (and user-selected subarrays).  Therefore, it is still possible to have a triggered event that contains e.g. 1 MST, if there were other MSTs that are not in the currently selected subarray that were triggered.  This  component can also help to simulate a more realistic array trigger, which would reject those events.  But normally, a multiplicity cut of 2 at the reconstruction level is sufficient (but at DL1 those events would still remain, unrealistically). That in particular does not work for the LSTs, as you could get a multiplicity-2 event with  1 LST+1 MST that would be rejected by the real LST trigger. Therefore, having a min multiplicity per type is needed.